batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/service/StudentService.java

```diff
@@ -25,8 +29,10 @@ public int fetchTotalCount() throws DataException {
     return studentDAO.fetchTotalCount();
   }
 
-  public void edit(Student student) throws DataException {
-    studentDAO.edit(student);
+  public void edit(Student student) throws DataException, ValidationExceptions {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)